Base/home/anon/tests/run-tests-and-shutdown.sh

```diff
@@ -6,26 +6,49 @@ echo "==== Running Tests on SerenityOS ===="
 run(index) {
     shift
     test_cmd=($*)
-    echo "Running test $index out of $count_of_all_tests -- $test_cmd"
+    echo "==== Running test $index out of $count_of_all_tests -- $test_cmd ===="
+    echo "::group::$test_cmd"
     if $test_cmd {
-        echo "::debug file=$test_cmd:: $test_cmd passed!"
+        echo "::endgroup::"
+        echo "==== $test_cmd passed! ===="
     } else {
+        echo "::endgroup::"
+        failed_tests=${concat_lists $failed_tests ($test_cmd)}
+        echo "==== Added $test_cmd to list of failed tests. Failed tests so far: $failed_tests ===="
         echo "::error file=$test_cmd:: $test_cmd returned non-zero exit code, check logs!"
     }
 }
 
-# TODO: test-web requires the window server
-system_tests=((test-js --show-progress=false) test-pthread /usr/Tests/LibM/test-math (test-crypto -t bigint))
-# FIXME: Running too much at once is likely to run into #5541. Remove commented out find below when stable
-all_tests=${concat_lists $system_tests} #$(find /usr/Tests -type f | grep -v Kernel | grep -v .inc | shuf))
+# Files in /usr/Tests/* that we don't want to execute match these patterns:
+#     Kernel: Kernel tests that are either old patched exploits, or hang the system/shell
+#     .inc: Shell test helper file that's not a test
+#     UserEmulator: Tests designed to run inside the Userspace Emulator
+#     stack-smash: Intentionally crashes by smashing the stack
+#     TestJSON: AK/TestJSON makes assumptions about $PWD to load its input files
```

Comment:
I think TestJSON has been fixed in the time since this was originally sent out.

Reply:
Hmm, I just ran it from /home/anon and it failed the same assertion as always. It works if you `cd` to /usr/Tests/AK` though. So it still makes assumptions about $PWD. :/ Just, assumptions that are a bit more valid than they were a month ago. Feels like something to cleanup in the next round that adds more "test configuration" type features.

Reply:
I see.. I thought I remembered someone deleting the test that depends on the file as input, but I guess I was wrong.

Reply:
The dependency on the ancient 4chan.json dump file was removed, but that TestJSON test still uses `test.frm` in order to work.

```c++

TEST_CASE(load_form)
{
    FILE* fp = fopen("test.frm", "r");
    VERIFY(fp);

```